sdk/communication/azure-communication-phonenumbers/src/test/java/com/azure/communication/phonenumbers/PhoneNumbersAsyncClientIntegrationTest.java

```diff
@@ -110,6 +113,9 @@ public void beginSearchAvailablePhoneNumbers(HttpClient httpClient) {
 
     @ParameterizedTest
     @MethodSource("com.azure.core.test.TestBase#getHttpClients")
+    @DisabledIfEnvironmentVariable(
+        named = "SKIP_LIVE_TEST",
+        matches = "(?i)(true)")
     public void beginSearchAvailablePhoneNumbersWithoutOptions(HttpClient httpClient) {
```

Comment:
Do we need to disable this test for sync as well?

Reply:
This test doesn't exist in the sync test file